source/includes/usage-examples/sample-app.cpp

```diff
@@ -0,0 +1,32 @@
+#include <iostream>
+
+#include <bsoncxx/builder/basic/document.hpp>
+#include <bsoncxx/json.hpp>
+#include <mongocxx/client.hpp>
```

Comment:
```suggestion
#include <mongocxx/client.hpp>
#include <mongocxx/exception/exception.hpp>
```

Add required include for `mongocxx::exception`.